cmd/common-main.go

```diff
@@ -37,19 +37,9 @@ func checkUpdate(mode string) {
 	}
 }
 
-func enableLoggers() {
-	fileLogTarget := globalServerConfig.Logger.GetFile()
-	if fileLogTarget.Enable {
-		err := InitFileLogger(&fileLogTarget)
-		fatalIf(err, "Unable to initialize file logger")
-		log.AddTarget(fileLogTarget)
-	}
-
-	consoleLogTarget := globalServerConfig.Logger.GetConsole()
-	if consoleLogTarget.Enable {
-		InitConsoleLogger(&consoleLogTarget)
-	}
-
+func enableConsoleLogger() {
```

Comment:
1. If console is only logger, nothing to enable i.e. its already enabled in `logger.go`
2. Aren't you removing `logrus` usage?

Reply:
I am not removing logrus usage in this PR.